OWNERS

```diff
@@ -0,0 +1,11 @@
+# approval == this is a good idea /approve
+approvers:
+  - humblec
+  - Madhu-1
+  - ShyamsundarR
```

Comment:
I think as it stands now, we should include "nixpanic" in the approvers list as well, unless nixpanic does not agree.

Reply:
Sure. Added @nixpanic to the owners and also updated the `reviewers` section as it stands today. :+1: